app/client/cypress/e2e/Regression/ClientSide/ActionExecution/ActionSelector_DownloadFunctions_spec.ts

```
@@ -0,0 +1,693 @@
+      deployMode.NavigateBacktoEditor();
+      EditorNavigation.SelectEntityByName("Button1", EntityType.Widget);
+      agHelper.WaitUntilEleAppear(locators._jsToggle("onClick"));
+      propPane.ToggleJSMode("onClick", false);
+      agHelper.WaitUntilEleAppear(
+        propPane._actionCardByTitle("Execute a JS function"),
+      );
+      agHelper.GetNClick(propPane._actionCardByTitle("Execute a JS function"));
+      agHelper.GetNClick(propPane._actionSelectorDelete);
+    });
+
+    it.only("8. To verify if the download() function downloads query data as a CSV file.", () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)